integrations/qdrant/tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+import pytest
+from haystack.dataclasses import SparseEmbedding
+
+
+@pytest.fixture
```

Comment:
I think we can scope it by session? Any reason why you would want different vectors for each different test function?